<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGrassTile"/>
   <addaction name="actionConcreteTile"/>
   <addaction name="actionDirtTile"/>
   <addaction name="actionWaterTile"/>
   <addaction name="actionEmptyTile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionGrassTile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Desktop/grassTile.png</normaloff>../../../../../Desktop/grassTile.png</iconset>
   </property>
   <property name="text">
    <string>grassTile</string>
   </property>
  </action>
  <action name="actionConcreteTile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Desktop/concreteTile.png</normaloff>../../../../../Desktop/concreteTile.png</iconset>
   </property>
   <property name="text">
    <string>concreteTile</string>
   </property>
  </action>
  <action name="actionDirtTile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Desktop/dirtTile.png</normaloff>../../../../../Desktop/dirtTile.png</iconset>
   </property>
   <property name="text">
    <string>dirtTile</string>
   </property>
  </action>
  <action name="actionWaterTile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Desktop/waterTile.png</normaloff>../../../../../Desktop/waterTile.png</iconset>
   </property>
   <property name="text">
    <string>waterTile</string>
   </property>
  </action>
  <action name="actionEmptyTile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../../Desktop/emptyTile.png</normaloff>../../../../../Desktop/emptyTile.png</iconset>
   </property>
   <property name="text">
    <string>emptyTile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
